selinux/ceph-installer.fc

```diff
@@ -0,0 +1,11 @@
+/usr/bin/gunicorn_pecan		--	gen_context(system_u:object_r:ceph_installer_exec_t,s0)
```

Comment:
I just copied the policy from Lukas's github repo. The changes are not there, yet. I suppose that is because the systemd service files still run gunicorn and celery directly so it would render this policy moot if the the two binary files were not labelled as ceph_installer_exec_t.